src/builder/commands/copy.rs

```diff
@@ -85,6 +91,7 @@ impl Copy {
         .member("include_regex", V::Scalar::new().optional())
         .member("owner_uid", V::Numeric::new().min(0).optional())
         .member("owner_gid", V::Numeric::new().min(0).optional())
+        .member("umask", V::Numeric::new().min(0).max(0o777).optional())
```

Comment:
A thought experiment: Where will you put a notice that non-specifing umask makes `image-cache-url` partially useless?

I mean people need always specify umask, or otherwise they will complain that `image-cache-url` doesn't work, and most of the time it would be hard to diagnose the problem.